src/pages/workspace/components/ActivityDetails/ActivityLinks.tsx

```
@@ -225,6 +229,11 @@ const ActivityLinks = ({ viewMode, activityId }: ActivityLinksProps) => {
                 strong
                 ellipsis={{ rows: 2 }}
                 style={{ marginBottom: 5, fontSize: 13, lineHeight: 1.2 }}
+                onClick={() => {
+                  if (item.url || '#') {
+                    window.open(item.url, '_blank');
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)